data/_ui/tooltips.txt

```diff
@@ -667,10 +667,10 @@ tip "shield protection:"
 	`Protection against shield damage. If you add more than one outfit with this attribute, each additional one is less effective: a total value of 1 results in a 1 percent reduction in shield damage, while a total value of 11 only results in a 10 percent reduction.`
 
 tip "silent jumps:"
-	`Prevents a ship from making hyperdrive or jump drive sounds when entering or leaving a system.`
+	`Prevents a ship from emitting sounds when entering or leaving a system via hyperjump.`
 
 tip "silent scans:"
-	`Prevents a ship from generating scan sounds.`
+	`Prevents a ship from emitting sounds while scanning other ships.`
```

Comment:
obiter: I think "whilst" could be used here but like who actually bothers anymore.